sale_order_project/models/sale.py

```diff
@@ -25,30 +23,3 @@ def _compute_related_project_id(self):
         string='Project',
         compute='_compute_related_project_id'
     )
-
```

Comment:
You deleted all the methods from sale.order model. This is not a good practice since the wizard should use them.